### ISSUE TITLE
$ ligatures

### ISSUE THREAD

**Issue by pavelbraginskiy** [tonsky/FiraCode#556]
A few ligatures with `$` would be nice, in addition to `<$`, `<$>`, and `$>`:

Perl (and sometimes Ruby) commonly uses `$_`, `$/`, `$\`, `$<`, `$!`, `$.`
Many shells use `$(`, `$?`, `$#`, `$@`

**Comment by tonsky** [comment]
Yeah, great suggestion, but sometimes it’s hard to think of a way to draw a ligature that actually improves things: it needs to be clear, logical etc. I don’t mind the sequences, it’s just hard to think of a better graphics for them, and I don’t want to just add ligatures for the ligatures sake
